luigi_pipeline/seqr_gcnv_loading.py

```diff
@@ -13,6 +13,21 @@
 
 SAMPLE_ID_REGEX = r'(.+)_v\d+_Exome_(C|RP-)\d+$'
 
+EXPECTED_TYPES = {
+    "start": hl.tint32, 
+    "end": hl.tint32, 
+    "CN": hl.tint32,
+    "QS": hl.tint32, 
+    "defragmented": hl.tbool, 
+    "vaf": hl.tfloat64, 
+    "vac": hl.tint32,
+    "genes_any_overlap_totalExons": hl.tint32,
+    "genes_strict_overlap_totalExons": hl.tint32,
+    "no_ovl": hl.tbool, 
+    "strvctvre_score": hl.tfloat64,
+    "is_latest": hl.tbool
```

Comment:
If you pass in array fields for the gene list fields, will hail properly parse those as lists or no?

Reply:
Looks like no... but almost!

```
Error summary: JsonParseException: Unrecognized token 'ENSG00000283761': was expecting (JSON String, Number, Array, Object or token 'null', 'true' or 'false')
 at [Source: UNKNOWN; line: 1, column: 16]
```

I imagine if the fields were proper JSON arrays rather than csv strings we'd be fine!  This was a super idea though.

Reply:
oh well, worth a try